### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
There is no consistency here (but now there is)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,84 +5,100 @@
 decorator-transforms 2.2.2 (patch)
 
 #### :bug: Bug Fix
-* `decorator-transforms`
-  * [#35](https://github.com/ef4/decorator-transforms/pull/35) Remove unintentional change that was part of #33 ([@ef4](https://github.com/ef4))
+
+- `decorator-transforms`
```
Comment (mansona) [inline CHANGELOG.md:9]: I probably wouldn't have prettier apply to the changelog since it's auto-generated 🤔

Comment (NullVoxPopuli) [reply CHANGELOG.md:9]: rogerroger.gif

**Review by ef4 — APPROVED** [review]

**Comment by NullVoxPopuli** [comment]
lockfile ... absent? wat

**Comment by NullVoxPopuli** [comment]
ah, there is no pnpm version enforcement / guidance
https://github.com/ef4/decorator-transforms/pull/39

**Pull request merged**